templates/todo/api/java/src/main/resources/application.properties

```diff
@@ -3,11 +3,13 @@ server.port=3100
 spring.jackson.date-format=com.microsoft.azure.simpletodo.configuration.RFC3339DateFormat
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 
+# Temporally disabling auto-configuration for key vault
 # Import KeyVault secrets as properties
-spring.cloud.azure.keyvault.secret.property-sources[0].enabled=true
-spring.cloud.azure.keyvault.secret.property-sources[0].endpoint=${AZURE_KEY_VAULT_ENDPOINT}
+# spring.cloud.azure.keyvault.secret.property-sources[0].enabled=true
+# spring.cloud.azure.keyvault.secret.property-sources[0].endpoint=${AZURE_KEY_VAULT_ENDPOINT}
+custom.azure.keyvault.secret.endpoint=${AZURE_KEY_VAULT_ENDPOINT},AZURE-COSMOS-CONNECTION-STRING
```

Comment:
Should we rename this property to be specific to the property that we're referencing? Like `custom.azure.keyvault.secret.mongodb`? For example, what happens if we need to customize a "SQL connection string" for example?